libdnf/hy-iutil.c

```diff
@@ -531,6 +531,63 @@ what_downgrades(Pool *pool, Id pkg)
     return l;
 }
 
+char *
+pool_solvable_epoch_optional_2str(Pool *pool, const Solvable *s, gboolean with_epoch)
```

Comment:
The function can be static in hy-query.c. What do you think @jrohel (It returns not permanent string, therefore it has to be handled with care).

Reply:
I agree. Moving the function to hy-query.c is the simplest and easy solution. We don't need to make the function public now.